db/migrate/20241108165827_convert_obfuscation_settings_to_params.rb

```diff
@@ -0,0 +1,22 @@
+class ConvertObfuscationSettingsToParams < ActiveRecord::Migration[6.1]
+  def change
+    # Create the common parameters if they don't exist and set the default values to false
+    CommonParameter.find_or_create_by!(name: 'obfuscate_inventory_hostnames', value: false)
+    CommonParameter.find_or_create_by!(name: 'obfuscate_inventory_ips', value: false)
```

Comment:
```suggestion
    hostname_setting = CommonParameter.find_or_create_by!(name: 'obfuscate_inventory_hostnames', value: false)
    ip_setting = CommonParameter.find_or_create_by!(name: 'obfuscate_inventory_ips', value: false)
```

You can skip the `find_by`  later in the code